tools/depends/target/freetype2-nohb/Makefile

```diff
@@ -0,0 +1,44 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=freetype
+VERSION=2.10.2
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.xz
+
+# Freetype has a circular dependency with harfbuzz.
+# To enable harfbuzz support in freetype, we build this first to bootstrap harfbuff
+# and then build freetype2 again with harfbuzz support
+
+# configuration settings
+# force using internal libtool
+export LIBTOOL=builds/unix/libtool
+CONFIGURE=cp -f $(CONFIG_SUB) $(CONFIG_GUESS) builds/unix; \
+          ./configure --prefix=$(PREFIX) --disable-shared --with-png=no
```

Comment:
please add --with-harfbuzz=no

and --with-harfbuzz=yes for "normal" freetype
as well as -Dfreetype=enabled for harfbuzz